.goreleaser.yml

```diff
@@ -290,6 +291,7 @@ nfpms:
     bindir: "/opt/tyk-gateway"
     overrides:
       rpm:
+        file_name_template: '{{ .PackageName }}_{{ replace .Version "-" "~" }}-1.{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}'
```

Comment:
@alephnull the common `file_name_template` is for debian and we override that for rpm. Or should we avoid the common `file_name_template` altogether and add both in `overrides?